packages/at_onboarding_flutter/example/android/app/build.gradle

```diff
@@ -8,7 +8,7 @@ if (localPropertiesFile.exists()) {
 
 def flutterRoot = localProperties.getProperty('flutter.sdk')
 if (flutterRoot == null) {
-    throw new GradleException("Flutter SDK not found. Define location with flutter.sdk in the local.properties file.")
+    throw new RuntimeException("Flutter SDK not found. Define location with flutter.sdk in the local.properties file.")
```

Comment:
This should not be changed, see the default Flutter build.gradle:

![CleanShot 2023-06-06 at 14 31 48](https://github.com/atsign-foundation/at_widgets/assets/33691921/6a039c98-a132-4eab-bb50-9ca25950cd89)

Reply:
Reverted to GradleException